sql/core/src/main/java/org/apache/spark/sql/connector/write/V1WriteBuilder.java

```diff
@@ -33,6 +33,11 @@
  */
 @Unstable
 public interface V1WriteBuilder extends WriteBuilder {
```

Comment:
This is an unstable API, can we just remove it and only use `V1Write`?

Reply:
I think that should simplify the V1 fallback API. I'll update it, @cloud-fan.

Reply:
Got rid of `V1WriteBuilder` and tried to update docs too. Let me know if I missed places, @cloud-fan.